src/app/layout/spinner/spinner.service.ts

```diff
@@ -19,7 +19,7 @@ export class SpinnerService {
     this.techRecordsLoadingState$ = this.store.pipe(select(technicalRecordsLoadingState));
   }
 
-  private reduceLoadingStates$() {
+  private get reduceLoadingStates$() {
     return combineLatest([this.globalLoadingState$, this.testResultLoadingState$, this.techRecordsLoadingState$]).pipe(
       map((states) =>
```

Comment:
Can this be
states.some(s => s === true);

Reply:
ooh didn't think of that!